collector/receiver/telemetryapireceiver/receiver.go

```diff
@@ -123,6 +142,23 @@ func (r *telemetryAPIReceiver) httpHandler(w http.ResponseWriter, req *http.Requ
 		case string(telemetryapi.PlatformInitRuntimeDone):
 			r.logger.Info(fmt.Sprintf("Init end: %s", r.lastPlatformEndTime), zap.Any("event", el))
 			r.lastPlatformEndTime = el.Time
+		case string(telemetryapi.Function):
+			logStr, ok := el.Record.(string)
+			if !ok {
+				continue
+			}
+
+			l, err := r.createLogRecord(el.Time, logStr)
```

Comment:
Wouldn't it be better to batch these - to allocate a `NewLogs()` (and associated `Scope`, etc.) before looping over the events, and wait to `ConsumeLogs` after the loop is done?